### DISCUSSION TITLE
Chatroom

### DISCUSSION THREAD

**Discussion by lachlansneff** [google/xls#522] [General]
Does the XLS project have a public chatroom (e.g. Matrix, irc, etc)?

**Comment by cdleary** [comment]
Hi Lachlan, we just have this GitHub discussion forum and the mailing list at the moment. Since the GitHub discussion forum has been low volume we weren't looking for more mediums to track, but if you have reason to think that's _causal_ (because these GH discussions are lacking in some particular way) I could look into how difficult it would be to get something stood up, LMK!
